### DISCUSSION TITLE
What "speed improvements" to use ?

### DISCUSSION THREAD

**Discussion by a3nima** [lllyasviel/stable-diffusion-webui-forge#501] [Q&A] [answered]
`Hint: your device supports --pin-shared-memory for potential speed improvements.
Hint: your device supports --cuda-malloc for potential speed improvements.
Hint: your device supports --cuda-stream for potential speed improvements.`

Says the console :) What of these would you suggest to use? And are there any negative side effects ?

(Im using a 3090 and windows 10.)

**Comment by ostap667inbox** [comment, marked as answer]
Read the [README.md](https://github.com/lllyasviel/stable-diffusion-webui-forge/blob/main/README.md#forge-backend)
There's a description of what each option is for and what the negative effects might be.

**Reply by a3nima** [reply]
thanks! looks like these options are more viable for less VRAM Graphic Cards, so I wont use them.
